packages/schema/src/fragments/Card.gql

```diff
@@ -1,6 +1,11 @@
-fragment ContentHubResultItem on ContentHubResultItem {
-  title
+fragment CardItem on CardItem {
```

Comment:
Filename is `Card.gql` but first fragment is `CardItem` - is this for `Card` or `CardItem` ? Would be good to standardise on naming it one or the other ways. Unless this `Card.gql` will contain more fragments like `CardTitle` or something?